### PR TITLE
[stable/dokuwiki] Replace lusotycoon/apache-exporter with bitnami/apache-exporter

### PR DESCRIPTION
**Pull request by marcosbc**
Signed-off-by: Marcos Bjoerkelund <marcos@bitnami.com>

#### What this PR does / why we need it:

Bitnami now builds an Apache Exporter based on the Lusitaniae/apache_exporter project. This PR updates this chart to use that container image.

#### Which issue this PR fixes
None

#### Special notes for your reviewer:
None
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/dokuwiki/README.md [hunk 1]
```diff
@@ -101,8 +101,8 @@ The following table lists the configurable parameters of the DokuWiki chart and
 | `podAnnotations`                     | Pod annotations                                            | `{}`                                          |
 | `metrics.enabled`                    | Start a side-car prometheus exporter                       | `false`                                       |
 | `metrics.image.registry`             | Apache exporter image registry                             | `docker.io`                                   |
-| `metrics.image.repository`           | Apache exporter image name                                 | `lusotycoon/apache-exporter`                  |
-| `metrics.image.tag`                  | Apache exporter image tag                                  | `v0.5.0`                                      |
+| `metrics.image.repository`           | Apache exporter image name                                 | `bitnami/apache-exporter`                  |
+| `metrics.image.tag`                  | Apache exporter image tag                                  | `0.7.0-debian-9-r1`                                      |
```
Comment (andresbono) [inline stable/dokuwiki/README.md:105]: The default value of the tag shouldn't be hardcoded in the README

File: stable/dokuwiki/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: dokuwiki
-version: 5.0.0
+version: 5.0.1
```
Comment (tompizmor) [inline stable/dokuwiki/Chart.yaml:3]: Should be 5.1.0

**Review by tompizmor — CHANGES_REQUESTED** [review]

**Comment by andresbono** [comment]
This can be closed in favor of #15835.

**Pull request closed without merging**